dependencies.gradle

```diff
@@ -3,17 +3,17 @@ ext {
     versionMinor = 0
     versionFix = 3
 
-    minSdkVersionVersion = 16
+    minSdkVersionVersion = 23
```

Comment:
This is because currently sample app uses `SYSTEM_ALERT_WINDOW` and `WRITE_SETTINGS`. These values exists from Marshmallow so lint causes warnings about them.